### DISCUSSION TITLE
How to remove the black box displayed when the tab header is selected?

### DISCUSSION THREAD

**Discussion by wartim** [Megabit/Blazorise#2320] [Q&A] [answered]
I want to achieve the effect shown in the document:
https://blazorise.com/examples/tabs/basic/#pills-home

View through F12,I found that I need to put the tab in the  card/card head tags
But when the label is clicked by default, a rectangular black border is displayed. I want to remove it, but I can’t find any effective CSS style.

**Comment by stsrki** [comment]
I'm not sure what black box. I cannot see it. Here is the SS of the last version

![image](https://user-images.githubusercontent.com/900302/117551572-19fb5380-b047-11eb-9a6e-e2d94cd81395.png)

What version do you use? And what browser?

**Reply by wartim** [reply]
@stsrki 
chrome 88.0.4324.150


Is it related to the version of bootstrap and popper I quoted? I don’t want to use CDN. I went to the official website to download. I couldn’t find the download of bootstrap-4.5.0 and popper.js@1.16.0. So I downloaded the revised version: bootstrap- 4.5.3 and popper. js@1.16.1

![3](https://user-images.githubusercontent.com/6902982/117567623-67cb9680-b0ef-11eb-9655-c93dcafa497e.jpg)

![1](https://user-images.githubusercontent.com/6902982/117567509-2b983600-b0ef-11eb-8e94-821a4b4d1506.jpg)
![2](https://user-images.githubusercontent.com/6902982/117567511-2cc96300-b0ef-11eb-9c1e-70c414bb89da.jpg)

**Reply by stsrki** [reply]
The version you use should be fine I guess.

What do you have in `site.css`? In default Blazor template it can mess with Bootstrap styles. You can also inspect in devtools to see what exactly is adding the borders.

**Comment by wartim** [comment]
The "master" version downloaded a week ago should be 0.9.3.6, at least written
![5](https://user-images.githubusercontent.com/6902982/117571022-5ee2c100-b0ff-11eb-96f2-41a12c3c0322.jpg)
![7](https://user-images.githubusercontent.com/6902982/117571085-9cdfe500-b0ff-11eb-83fa-8b5f1bc4926a.jpg)

**Comment by wartim** [comment]
And after clicking the tab, a black focus box appears, the css just automatically add class: "active", and then if I click on the blank space of the page again, the black box disappears, and there is no change in the CSS style
![9](https://user-images.githubusercontent.com/6902982/117571451-3b207a80-b101-11eb-9b4c-793fbb258a02.jpg)
![12](https://user-images.githubusercontent.com/6902982/117571917-3230a880-b103-11eb-8bc7-969aecffca3d.jpg)

**Reply by stsrki** [reply]
I tried the same code as you have and I just cant reproduce the issue on my side.

**Comment by wartim** [comment]
And ctrl + f5 does not work, I removed the redundant code of the project and made a demo

[demo.zip](https://github.com/stsrki/Blazorise/files/6447602/demo.zip)

**Reply by stsrki** [reply]
Still no black box for me

![image](https://user-images.githubusercontent.com/900302/117574107-d4d53100-b0db-11eb-8386-07c620d82d8c.png)

**Comment by wartim** [comment]
Is it another browser version problem?
Google Chrome  88.0.4324.150

**Comment by wartim** [comment]
https://user-images.githubusercontent.com/6902982/117574514-fbad5a80-b10f-11eb-8b40-73d4491ce191.mp4

**Comment by wartim** [comment, marked as answer]
@stsrki 
I tested it out, and on my computer, I  have to add a css style so that there is no black box

![32](https://user-images.githubusercontent.com/6902982/117575094-f1409000-b112-11eb-8150-218be1bb6606.jpg)

**Reply by stsrki** [reply]
It might be browser version problem. Mine is 90.x
